web/source/kmwkeyboards.ts

```diff
@@ -356,6 +364,11 @@ class KeyboardManager {
     //TODO: This does not make sense: the callbacks should be in _SetActiveKeyboard, not here,
     //      since this is always called FROM the UI, which should not need notification.
     //      If UI callbacks are needed at all, they should be within _SetActiveKeyboard  
+
+    if(PInternalName && PInternalName.indexOf("Keyboard_") != 0) {
```

Comment:
Surely that test should be `< 0`?

Reply:
I considered that, but decided that on the off-chance someone made a keyboard named, say, `My_Keyboard_For_Work`, we'd still want it to properly prepend the `Keyboard_` internal prefix on the method.